### ISSUE TITLE
Mobile Friendly Version of website.

### ISSUE THREAD

**Issue by 0xPrateek** [0xPrateek/Portfolio-Template#8]
- Currently, our portfolio-template only supports the desktop view.
This Issue is for creating a mobile version of site.

**Comment by angelmtztrc** [comment]
Can I work on that?

**Comment by 0xPrateek** [comment]
It would be great. I'll assign you this issue.

**Comment by Faizan-999** [comment]
hey
can i work your project ? i want to solve your issue

**Comment by 0xPrateek** [comment]
Yes sure..! Just comment on an Issue that you want to work on I'll assign it to you.

**Comment by codetronaut** [comment]
Hey, @0xPrateek I used your template and it was good, 
Minor improvement: In Readme you should remove the `GPL` Licence as you have added the `MIT` one.
